scilpy/image/operations.py

```diff
@@ -0,0 +1,461 @@
+# -*- coding: utf-8 -*-
+
+"""
+Utility operations provided for scil_image_math.py and scil_connectiity_math.py
+They basically act as wrappers around numpy to avoid installing MRtrix/FSL
+to apply simple operations on nibabel images or numpy arrays.
+"""
+
+from copy import copy
+import logging
+
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter
+from scipy.ndimage.morphology import (binary_closing, binary_dilation,
+                                      binary_erosion, binary_opening)
+
+from scilpy.utils.util import is_float
+
+
+def get_array_ops():
+    """Get a dictionary of all functions relating to array operations"""
+    return {
+        'lower_threshold': lower_threshold,
+        'upper_threshold': upper_threshold,
+        'lower_clip': lower_clip,
+        'upper_clip': upper_clip,
+        'absolute_value': absolute_value,
+        'round': around,
+        'ceil': ceil,
+        'floor': floor,
+        'normalize_sum': normalize_sum,
+        'normalize_max': normalize_max,
+        'convert': convert,
+        'invert': invert,
+        'addition': addition,
+        'subtraction': subtraction,
+        'multiplication': multiplication,
+        'division': division,
+        'mean': mean,
+        'std': std,
+        'union': union,
+        'intersection': intersection,
+        'difference': difference,
+    }
+
+
+def get_image_ops():
+    """Get a dictionary of all functions relating to image operations"""
+    image_ops = get_array_ops()
+    image_ops.update({
+        'dilation': dilation,
+        'erosion': erosion,
+        'closing': closing,
+        'opening': opening,
+        'blur': gaussian_blur
+    })
+    return image_ops
+
+
+def get_operations_doc(ops: dict):
+    """From a dictionary mapping operation names to functions, fetch and join
+    all documentations, using the provided names."""
+    full_doc = []
+    for func in ops.values():
+        full_doc.append(func.__doc__)
+    return "".join(full_doc)
+
+
+def get_array_operations_doc():
+    """Fetch documentation from all array operations."""
+    return "".join([f.__doc__ for f in get_array_ops().values()])
```

Comment:
Hum... Not sure I understand why you don't use get_operations_doc. Couldn't you make it to deal with it with something like:

get_array_operations_doc():
        get_operations_doc(get_array_ops())

Or else can you show better the similarity between the two functions? Ex: func vs f. Ex, create it first and then just return "".join(full_doc)?